node/src/service.rs

```diff
@@ -194,27 +193,22 @@ where
 }
 
 /// Build the import queue for the template runtime (aura + grandpa).
-pub fn build_babe_grandpa_import_queue<RuntimeApi, Executor>(
-    client: Arc<FullClient<RuntimeApi, Executor>>,
+pub fn build_babe_grandpa_import_queue(
+    client: Arc<FullClient>,
     config: &Configuration,
     eth_config: &EthConfiguration,
     task_manager: &TaskManager,
     telemetry: Option<TelemetryHandle>,
-    grandpa_block_import: GrandpaBlockImport<FullClient<RuntimeApi, Executor>>,
+    grandpa_block_import: GrandpaBlockImport<FullClient>,
     select_chain: FullSelectChain,
     offchain_tx_pool_factory: OffchainTransactionPoolFactory<Block>,
 ) -> Result<
-    ((BasicImportQueue, BabeWorkerHandle<Block>), BoxBlockImport, BabeLink<Block>),
+    ((BasicImportQueue, BabeWorkerHandle<Block>), BoxBlockImport<Block>, BabeLink<Block>),
     ServiceError,
->
-where
-    RuntimeApi: ConstructRuntimeApi<Block, FullClient<RuntimeApi, Executor>>,
-    RuntimeApi: Send + Sync + 'static,
-    RuntimeApi::RuntimeApi: RuntimeApiCollection,
-    Executor: NativeExecutionDispatch + 'static,
-{
-    let _frontier_block_import =
-        FrontierBlockImport::new(grandpa_block_import.clone(), client.clone());
+> {
+    // TODO should we use this instead of babe block import?
+    // let _frontier_block_import =
```

Comment:
This code creates a new instance of the Frontier block importer, which is used to support Ethereum-compatible functionality. The importer integrates with the existing GRANDPA consensus mechanism using grandpa_block_import and client. 
If we remove this import, are we sure that Ethereum transactions and smart contracts will be supported?

Reply:
Please, check further. It has been abandoned for a while. It’s not use in the code - babe import queue is used.

Reply:
The code wouldn’t compile if I randomly disabled variable which used somewhere later